service/src/test/java/uk/gov/hmcts/reform/fpl/utils/elasticsearch/ESQueryTest.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.fpl.utils.elasticsearch;
+
+import org.json.JSONObject;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ESQueryTest {
+
+    @Test
+    void shouldWrapWithQueryField() {
+        ESQuery query = new TestClass();
+
+        final JSONObject queryInContext = query.toQueryContext();
+        final JSONObject expectedContext = new JSONObject(Map.of("query", Map.of()));
+
+        assertThat(queryInContext).usingRecursiveComparison().isEqualTo(expectedContext);
+    }
+
+    @Test
+    void shouldAddSizeFieldWhenProvided() {
+        ESQuery query = new TestClass();
+
+        final JSONObject queryInContext = query.toQueryContext(2);
+        final JSONObject expectedContext = new JSONObject(Map.of("query", Map.of(), "size", 2));
+
+        assertThat(queryInContext).usingRecursiveComparison().isEqualTo(expectedContext);
+    }
+
+    private static class TestClass implements ESQuery {
+        @Override
+        public Map<String, Object> toMap() {
+            return Map.of();
```

Comment:
it would be better to have some values in a map, otherwise in test when we see
final JSONObject expectedContext = new JSONObject(Map.of("query", Map.of(), "size", 2));
then cannot be sure that Map.of() is some kind of default value or value from test object